todo_app/gunicorn.conf.py

```diff
@@ -1,4 +1,5 @@
-bind = "0.0.0.0:5000"
+import os
+bind = "0.0.0.0:" + os.environ.get('PORT')
```

Comment:
Note that here we don't have a default argument for PORT, which means that your README instructions for your prod container currently fail locally as the user doesn't provide that automaticallly. We should ensure that the guidance is still valid; the easiest way would likely be to provide a default PORT argument, which we could do in your dockerfile or here (e.g. update this to `bind = "0.0.0.0:" + os.environ.get('PORT', "5000")`)

Reply:
Hi, 
I did foresee this actually and I added a PORT=5000 to my .env file - admittedly I didn't add this to the .env.template or the readme file! Would this have worked? I see how adding a default port to the argument is a more elegant solution: I'll update my code.
Thanks